Flour_Power/mod.json

```diff
@@ -1,6 +1,6 @@
 {
   "mod_id": "Flour_Power",
-  "mod_name": "FlourPower",
+  "mod_name": "Flour Power",
   "mod_dir": "/FlourPower/",

```

Comment:
Hello,
The ``mod_dir`` need to match the dir where the texutres are stored. In this it has to be "mod_dir": "/Flour_Power/",